app/controllers/projects_controller.rb

```diff
@@ -69,16 +69,23 @@ def index
         sample_count_by_project_id = Hash[samples.group_by(&:project_id).map { |k, v| [k, v.count] }]
         host_genome_names_by_project_id = {}
         tissues_by_project_id = {}
-        samples.includes(:host_genome).each do |s|
-          (host_genome_names_by_project_id[s.project_id] ||= Set.new) << s.host_genome.name if s.host_genome && s.host_genome.name
+        min_sample_by_project_id = {}
+        owner_by_project_id = {}
+        samples.includes(:host_genome, :user).each do |s|
+          (host_genome_names_by_project_id[s.project_id] ||= Set.new) << s.host_genome.name if s.host_genome.name
           # TODO: sample_tissue column is deprecated, retrieve sample_type from Metadatum model instead
           (tissues_by_project_id[s.project_id] ||= Set.new) << s.sample_tissue if s.sample_tissue
+          if !min_sample_by_project_id[s.project_id] || min_sample_by_project_id[s.project_id] < s.id
```

Comment:
why is it important to get the min sample? are the sample owners not always the same within a project? does the first sample have a special status?

Reply:
No. Anyone associated with the project can upload.

Reply:
Btw, we should have project owner explicitly set... but that is not the case right now.

Reply:
Might be worth extracting the assignment of an owner into a function and/or commenting very explicitly, since we might want to change the ownership function in the future?

Reply:
I will go with comment for now.
I think what you say is actually valid for tissues and host_genomes. All three queries need to be refactored.